src/main.js

```diff
@@ -100,11 +100,6 @@ ReactDOM.render(App, container)
 // Sets the splash to hidden when the page is rendered
 document.getElementById('splash').className += ' splash-hidden'
 
-if (navigator.serviceWorker) {
-  navigator.serviceWorker.register('/sw.js', {scope: '/'})
-    .catch((err) => console.error('Unable to register service worker.', err))
-}
```

Comment:
Maybe we need to unregister the current ServiceWorker... Otherwise the service worker may still be in place. 🤔

Reply:
I think sw.js is no longer available to it should fail to start

Reply:
I think sw.js is probably cached by the service worker itself :D

Reply:
https://stackoverflow.com/questions/33704791/how-do-i-uninstall-a-service-worker

Reply:
lool you could be right 😭 
k Ill add code to unregister all

Reply:
done